test/models/coupon_test.rb

```diff
@@ -1,7 +1,22 @@
 require 'test_helper'
 
 class CouponTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'coupon in valid state' do
+    coupon = create(:coupon)
+    assert coupon.valid?
```

Comment:
I believe this test just tests that factory girl is making the coupon properly, and it would be better to manually pass in good params and assert that that coupon saves and is valid.